README.md

```diff
@@ -256,3 +256,263 @@ For example, to run the `scale` command using Kourier,
+     50%   40.62ms
+     75%   42.76ms
+     90%   59.33ms
+     99%   73.85ms
+  24447 requests in 1.00m, 4.33MB read
+  Socket errors: connect 0, read 0, write 0, timeout 30
+Requests/sec:    406.82
+Transfer/sec:     73.86KB
+
+[Verbose] Replicas scale from 0 to 5:
+replica_count   ready_duration(seconds)
+            0                    31.974
+            1                    41.548
+            2                    46.477
+            3                    46.836
+            4                    47.841
+
+[Verbose] Pods scale from 0 to 5:
+pod_count       ready_duration(seconds)
+        0                          35.0
+        1                          25.0
+        2                          35.0
+        3                          32.0
+        4                          30.0
+----------------------------------------------------------------------------------------
+[Verbose] Namespace ktest, Service ktest-3:
+[Verbose] wrk output:
+Running 1m test @ http://192.168.0.181:32283
+  2 threads and 30 connections
+  Thread Stats   Avg      Stdev     Max   +/- Stdev
+    Latency    37.48ms   16.49ms 234.53ms   66.06%
+    Req/Sec   400.96    183.79     3.42k    94.26%
+  Latency Distribution
+     50%   40.36ms
+     75%   42.13ms
+     90%   58.91ms
+     99%   73.41ms
+  27208 requests in 1.00m, 4.82MB read
+  Socket errors: connect 0, read 0, write 0, timeout 30
+Requests/sec:    452.74
+Transfer/sec:     82.19KB
+
+[Verbose] Replicas scale from 0 to 5:
+replica_count   ready_duration(seconds)
+            0                    31.631
+            1                    37.388
+            2                    41.688
+            3                    43.778
+            4                    45.875
+
+[Verbose] Pods scale from 0 to 5:
+pod_count       ready_duration(seconds)
+        0                          35.0
+        1                          31.0
+        2                          33.0
+        3                          24.0
+        4                          35.0
+----------------------------------------------------------------------------------------
+Measurement saved in CSV file /tmp/20220601083752_ksvc_loading_time.csv
+Measurement saved in JSON file /tmp/20220601083752_ksvc_loading_time.json
+Visualized measurement saved in HTML file /tmp/20220601083752_ksvc_loading_time.html
+```
+
+- CSV result
+
+```bash
+$ cat /tmp/20220602084358_ksvc_loading_time.csv
+svc_name,svc_namespace,replica_1_ready,replica_2_ready,replica_3_ready,replica_4_ready,replica_5_ready,replica_6_ready,replica_7_ready
+ktest-0,ktest,47.002,48.574,52.243,53.943,55.452,58.782,59.586
+ktest-1,ktest,45.176,48.184,50.566,50.902,51.609,54.727,58.331
+ktest-2,ktest,38.084,44.077,44.279,46.016,47.625,59.914,60.824
+ktest-3,ktest,44.560,47.732,49.989,52.944,56.772,57.884,59.263
+ktest-4,ktest,46.383,48.887,49.199,49.907,56.195,57.475,60.374
+```
+
+- HTML result in dashboard
+
+![service_load_replicas_latency measurement](docs/service_load_replicas_latency.png)
```

Comment:
Please add a new line at the end in order to fix a style/lint check failure

Reply:
The chart in the PNG file is interesting. The X axis are services. Each line shows the changes from service1 to service5 for a specific replica number. I just wonder if it's meaningful. 
Maybe as a user, I want to see a line to show the changes from replica 1 to replica 5 for a single service. In order to show such lines, we should use replica numbers as X axis.
I just put an open question here. Which chat is better? to show changes from service1 to service5 for a specific replica number (the current solution) or to show the changes from replica 1 to replica 5 for a single service (my proposed solution) ?

Reply:
Thanks for your suggestion, I can add another CSV and HTML result to show the change line from replica 1 to replica N for each service in the follow-up PR.